README.md

```diff
@@ -26,3 +26,21 @@ Debian build facilities are integrated into the repo at `debian/`.
 3. Install the resulting `jellyfin*.deb` file on your system.
 
 A huge thanks to Carlos Hernandez who created the Debian build configuration for Emby 3.1.1.
+
+### Windows (64 bit)
+A pre-built windows installer will be available at [The JellyFin Repository](https://repo.jellyfin.org/).
+
+1. Install the dotnet core SDK 2.1 from [Microsoft's Webpage](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-2.1.500-windows-x64-installer)
+2. Clone Jellyfin into a directory of your choice. From that directory run in powershell `dotnet publish -c Release -r win10-x64 MediaBrowser.sln -o $Env:APPDATA\Jellyfin-Server` or in CMD `dotnet publish -c Release -r win10-x64 MediaBrowser.sln -o %APPDATA%\Jellyfin-Server`
+3. (Optional) Copy the ffmpeg binaries into the Jellyfin directory:
```

Comment:
Is it worth moving this script into a file in the Jellyfin repository? That would make the instructions here slightly easier to follow and means this script could be source controlled, too.

Reply:
It is, and we should. Is there a logical place in the jellyfin repo for build scripts? If so I agree it'd make sense to put it there.

Reply:
Honestly just throw it in the root with a name like `build-windows.ps` or something. All the others (e.g. the Dockerfile, the `debian` directory) are all just relative to the repo root. We can clean it up/move it around eventually if needed.

Reply:
K. Maybe reject this pull request for now and I will do the rebase against current master and include a build script in a new one

Reply:
It's fine, just force-push to your branch :-)

Reply:
I added a WIP tag just now for this sort of situation, feel free to add it while you work/force push and we can go from there!